Tests/Unit/Extend/Controller/Admin/PaymentCountryTest.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+use Wirecard\Oxid\Extend\Controller\Admin\PaymentCountry;
+
+class PaymentCountryTest extends \OxidEsales\TestingLibrary\UnitTestCase
+{
+    /**
+     * @var PaymentCountry
+     */
+    private $oPaymentCountry;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->oPaymentCountry = oxNew(PaymentCountry::class);
+    }
+
+    /**
+     * @dataProvider renderProvider
+     */
+    public function testRender($sPaymentId, $bAllowCountryAssignment)
+    {
+        $_POST['oxid'] = $sPaymentId;
```

Comment:
there is a `$this->setRequestParameter()` function in the Oxid `UnitTestCase` that does exactly this. maybe we should use that to be more consistent what the framework does.

Reply:
👍